src/utils/genes.js

```diff
@@ -0,0 +1,19 @@
+const upperCaseArray = (array) => (array.map((element) => element.toUpperCase()));
+
+const findLoadedGenes = (matrix, selectedGenes) => {
```

Comment:
Maybe it is a general question about the structure and ordering of the files in `utils`, but if in the future someone needs to use the same functionality as the one implemented in `findLoadedGenes`, how will they find that the functionality already exists before reimplementing it themselves?

Reply:
If this is something that is very specific to `loadGeneExpression` action, then is it necessary to create a new file in utils for it, instead of just having the function in the `loadGeneExpression` file?

Reply:
No idea, it's a good question for utils in general I think.

My heuristic on this is:
- I need to do A.
- Look at places where I know A/something similar is being done.
- If I can reuse what is being done, abstract it and reuse it. If there is a utils function I see being used, I reuse it.

Reply:
```
My heuristic on this is:

I need to do A.
Look at places where I know A/something similar is being done.
If I can reuse what is being done with minimal work, abstract it and reuse it. If there is a utils function I see being used, I reuse it.
```

Sounds like a good heuristic. My concern here is that this `findLoadedGenes` is super specific to the `loadGeneExpression` action. How likely is it that someone might want to reuse it in the future?

Reply:
good point, I had moved it before when I had thought to use it in both loadgeneexpression and loaddownsampledgeneexpression, but I think it would just be back in the loadgeneexpression file only

Reply:
I'll resolve this when it is back there

Reply:
Moved `findLoadedGenes` to `loadGeneExpression` and changed the utils file to export default `uppercaseArray` and be called `upperCaseArray.js`